python-libraries/data-platform-catalogue/data_platform_catalogue/client/datahub/datahub_client.py

```diff
@@ -232,7 +242,7 @@ def upsert_table(
             customProperties={
                 "sourceDatasetName": metadata.source_dataset_name,
                 "sourceDatasetLocation": metadata.source_dataset_location,
-                "sensitivityLevel": metadata.data_sensitivity_level,
+                "sensitivityLevel": str(metadata.data_sensitivity_level),
                 "rowCount": "1177",
```

Comment:
We've renamed source dataset location so may as well add that in here
```suggestion
                "whereToAccessDataset": "Analytical Platform",
                "sensitivityLevel": metadata.data_sensitivity_level.name,
                "rowCount": "1177",

Reply:
Included

Reply:
I'd keep `whereToAccessDataset` as `"metadata.where_to_access_dataset` as you've specified in `entities.py`.

I'd provide "Analytical Platform" as a default suggestion in a UI dropdown, but I'm wary of adding it here as we don't know what other data people may add